CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## 0.0.1 -
+
+### Added
+  - lib/schemas/payment.js with validators for processing and creating payments.
```

Comment:
No need to indent these.

Reply:
fixed: https://github.com/digitalbazaar/bedrock-payment-http/pull/1/commits/15b56f878356ae8fe87cb90dee66017a31345754